play

```diff
@@ -80,13 +80,23 @@ sudoku()            { runghc "$1"; }
 tsp()               { runghc "$1"; }
 
 # Help
-printTitle() {
+printFileWithTitle() {
     printf '%0.1s' "="{1..80};echo
     echo "= " "$1"
     printf '%0.1s' "="{1..80};echo
+    cat "$1"
 }
 gamecat()  { echo "$1" | cut -d/ -f1; }
 gamename() { basename "$1" .hs; }
+detectAndWarnAboutFreeglut() {
+    # NOTE: The detection logic requires gcc, without it will produce false alarm.
+    #       But can you imagine the life of people without gcc?
```

Comment:
Yes, I think I can imagine linux systems which don't yet have dev tools like gcc installed. There's nothing worse than misleading error messages, so if we must use gcc here I think it's better to first check for that and give a more specific message.